packages/react-magma-dom/src/components/Tooltip/index.tsx

```diff
@@ -72,7 +75,7 @@ const ToolTipArrow = styled.span<{ position?: any; isInverse?: boolean }>`
   }
 `;
 
-const StyledTooltip = styled.div<{
+export const StyledTooltip = styled.div<{
```

Comment:
we're doing this for `StyledButton` too where `@react-magma/charts` is using the internal version... should we be?